.github/workflows/release.yml

```diff
@@ -38,10 +38,10 @@ jobs:
           ref: ${{ github.ref }}
           clean: true
       - name: "PDK Build"
-        uses: docker://puppet/pdk:nightly
+        uses: docker://puppet/puppet/pdk:2.6.1.0
```

Comment:
```suggestion
        uses: docker://puppet/pdk:2.6.1.0
```

Reply:
small typo

Reply:
@MartyEwings can it also be `docker://puppet/pdk:latest` or better to pin it? Since this is now unmanaged, this will mean more maintenance...